examples/crab-saber/Cargo.toml

```diff
@@ -19,7 +19,7 @@ rand = "0.8.0"
 approx = "0.5"
 
 [target.'cfg(target_os = "android")'.dependencies]
-ndk-glue = "=0.6.0"
+ndk-glue = "0.6"
```

Comment:
There's historical reasons for this, unfortunately.

We *MUST* keep our `ndk-glue` in lockstep with the rest of the `ndk-glue` ecosystem. Several upstream packages, such as `openxr` and `cpal` will break if the `ndk-glue` version gets out of sync.

I'd prefer to leave this as is for now, and approach this by switching to `ndk-context` as documented here:

https://github.com/rust-windowing/android-ndk-rs/issues/211

Reply:
Doesn't cargo's semver resolution solve this? My understanding is that compatible versions of libraries (i.e. 0.6.x) are required to resolve to one specific version. The problem that this is intended to fix is that `openxr` [no longer depends on `ndk-glue`, but depends on `ndk-context` instead](https://github.com/Ralith/openxrs/commit/556b7d83cba5e7eac737bb10ea6c1b499377d66f), but an application using `hotham` can't force the version of `ndk-glue` to 0.6.2, which is the version that starts using `ndk-context`.

Reply:
re: semver resolution, no. I thought the same thing but was unpleasantly surprised when things blew up my face on a fresh build when something in the `ndk-glue` ecosystem had been updated. To be clear, we could still get bitten by other dependencies but this is still a start.

re: `openxr` and `ndk-context` can you expand on that a bit further?

Reply:
When we update to the next version of `openxr` (or in my case, if you use `[patches.crates-io]` to move to a top of tree version to get access to `XR_FB_display_refresh_rate`), things will break, because `openxr` is removed its dependency on `ndk-glue` and added one on `ndk-context` instead.

In version 0.6.2, `ndk-glue` moves its global state into `ndk-context` and properly initializes `ndk-context`, so if a sufficiently new version of `ndk-glue` is used for initialization, crates that depend on `ndk-context` will also work. However, we're forcing the version to exactly 0.6.0, so the result is that hotham applications will have use the old `ndk-glue` that doesn't initialize `ndk-context`, and a new version of `openxr` will use an uninitialized `ndk-context` and immediately crash.

(BTW, we're *not* in lockstep with the rest of the ndk-glue ecosystem: [`cpal`] uses `0.6`, and `openxr` [used](https://github.com/Ralith/openxrs/blob/29cb8293db5ad48fd40e69e8ee23d452608b5438/openxr/Cargo.toml#L36) `0.6`, and effectively will be `>0.6.2`)

Reply:
Completely agree @jmgao - moving to `ndk-context` is the way to go. I'd just like to approach that in a separate PR. 😄